cobc/cobc.c

```diff
@@ -6533,16 +6481,103 @@ line_has_listing_statement (char *line, const enum cb_format source_format)
 	return 1;
 }
 
+static char *
+get_word (char *str)
+{
+	int i = 0;
+
+	if (!str) {
+		return NULL;
+	}
+	while (str[i] && isspace (str[i])) {
+		++i;
+	}
+	if (str[i]) {
+		return str + i;
+	}
+	return NULL;
+}
+
+static char *
+get_next_word (char *str)
+{
+	int i = 0;
+
+	if (!str) {
+		return NULL;
+	}
+	while (str[i] && !isspace (str[i])) {
+		++i;
+	}
+	while (str[i] && isspace (str[i])) {
+		++i;
+	}
+	if (str[i]) {
+		return str + i;
+	}
+	return NULL;
+}
+
+
+static COB_INLINE COB_A_INLINE int
+is_debug_line (char *line, int fixed, int acudebug)
+{
+	if (line == NULL || line[0] == 0) {
+		return 0;
+	}
+	return !cb_flag_debugging_line
+		&& ((fixed && line[cobc_get_indicator ()] == 'D')
+		    || (!fixed && (acudebug
+				   ? !strncasecmp (line, "\\D", 2)
+				   : !strncasecmp (line, "D ", 2))));
+}
+
+static COB_INLINE COB_A_INLINE int
+is_comment_line (char *line, int fixed)
+{
+	if (line == NULL || line[0] == 0) {
+		return 0;
+	} else {
+		const int indicator = cobc_get_indicator ();
+		return fixed
+			? line[indicator] == '*' || line[indicator] == '/'
+			: !strncmp (line, "*>", 2);
+	}
+}
+
+static void
```

Comment:
This should have already been `static COB_INLINE COB_A_INLINE void`, the new below should be, too - please adjust both.

Reply:
Corrected but I don't understand when I should use this: COB_INLINE COB_A_INLINE ? Maybe when the function is often called ?

Reply:
yes, when the function is often called and small